src/mjolnir/ingest_transit.cc

```diff
@@ -497,6 +517,7 @@ bool write_stop_pair(Transit& tile,
 
       // get frequency info
       if (!feed.get_frequencies(currTrip->trip_id).empty()) {
+        // TODO(nils): is this correct? a trip has a unique frequency?
```

Comment:
assumption nope

Reply:
this is actually not true: even the Google example feed has multiple frequencies for the same trip id. I'll re-add the comment